src/super_gradients/training/utils/checkpoint_utils.py

```diff
@@ -206,10 +217,6 @@ def load_checkpoint_to_model(
     if isinstance(strict, str):
         strict = StrictLoad(strict)
 
-    if ckpt_local_path is None or not os.path.exists(ckpt_local_path):
-        error_msg = "Error - loading Model Checkpoint: Path {} does not exist".format(ckpt_local_path)
-        raise RuntimeError(error_msg)
-
```

Comment:
Why remove this ? Because we check it before already?

Reply:
We have exactly the same check inside the method itself, yes.
But since we are adding URL support, this one triggers as false-positive

Reply:
Yeah that's right